connector/client/survivalanalysis/cli.go

```diff
@@ -158,7 +158,12 @@ func ExecuteClientSurvival(token, parameterFileURL, username, password string, d
 	// --- convert panels
 	timer := time.Now()
 	logrus.Info("converting panels")
```

Comment:
Even if not from this change, the logs here need more context, e.g. "Survival analysis <ID>: ..." 

Remember that when looking at the logs in a console everything is mixed up together. This makes debugging easier.